### ISSUE TITLE
imgdrivecmd_2.1.9_x64.exe cannot specify drive letter.

### ISSUE THREAD

**Issue by Hiroki-H1974** [dvdforge/imgdrive-docs#1]
When I try to mount some image file to X: drive,
C:\exe\imgdrivecmd_2.1.9_x64.exe" -m:X "%~df1"
The result is mounted as D: Drive.
What is this? some problems or mistake on me?

**Comment by abdullah-erturk** [comment]
This problem persists in the latest version. When trying to use a specific drive letter via the command line, that drive letter is not used.

I hope this problem will be resolved.

**Comment by imgdrive** [comment]
Please make sure the specific drive letter exists, and is a virtual drive created by ImgDrive.

**Comment by Hiroki-H1974** [comment]
@imgdrive Is ImgDrive possible by command line to create virtual drives?

**Comment by imgdrive** [comment]
https://download.yubsoft.com/imgdrivecmd_2.2.6.1.exe

Added a -n switch to set the number of drives: imgdrivecmd_2.2.6.1.exe -n:2

**Comment by Hiroki-H1974** [comment]
@imgdrive Can commandline specify drive letter to create?

**Comment by imgdrive** [comment]
No, it's impossible to implement this feature with the storport driver.

**Comment by abdullah-erturk** [comment]
When I use the command `imgdrivecmd_2.2.6.1.exe -m:K D:\BOOTPE.iso`, unfortunately the K: drive letter is not used.

Instead of using the letter I want, imgdrive uses the next available free letter.
